core/src/test/java/org/apache/iceberg/TestIncrementalDataTableScan.java

```diff
@@ -31,49 +35,42 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Iterables;
 import org.apache.iceberg.relocated.com.google.common.collect.Lists;
 import org.apache.iceberg.relocated.com.google.common.collect.Sets;
-import org.assertj.core.api.Assertions;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-
-@RunWith(Parameterized.class)
-public class TestIncrementalDataTableScan extends TableTestBase {
-  @Parameterized.Parameters(name = "formatVersion = {0}")
-  public static Object[] parameters() {
-    return new Object[] {1, 2};
-  }
-
-  public TestIncrementalDataTableScan(int formatVersion) {
-    super(formatVersion);
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(ParameterizedTestExtension.class)
+public class TestIncrementalDataTableScan extends TestBase {
+  @Parameters(name = "formatVersion = {0}")
+  public static List<Object> parameters() {
```

Comment:
```suggestion
  protected static List<Object> parameters() {
```
this was most likely previously required to be public by JUnit4, but that's not the case anymore with JUnit5, so we can make all of the parameters() methods protected

Reply:
Okay sure, thanks for the advice.